app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,10 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">"Does it have drinking water?"</string>
```

Comment:
what in case where there is shop selling bottled water?

Reply:
If water was otherwise unavailable, I would leave a note and indicate a shop there in that case (so no, I don't think existence of a convenience shop qualifies as `drinking_water=yes`)
Perhaps `Does it have freely available drinking water` would be better question (even if longer)?

Reply:
As in other thread, I would assume all facilities on camp site are available only to those who have legally obtained access to that camp site (e.g. payed for their say if usual `fee=yes`). Anyway, would `Is drinking water available to all customers here?` sound better to you (akin to other thread question)? If not, feel free to suggest change for better wording.